### DISCUSSION TITLE
3000项目对象中类型为相关表的字段在编辑时有时回显的是id，5000项目显示的是名称，麻烦看下3000项目为什么会回显id

### DISCUSSION THREAD

**Discussion by daiyaozuo** [steedos/steedos-platform#4190] [General]
![image](https://user-images.githubusercontent.com/66349613/215368470-16f5883c-3533-4d9d-b2ec-959b8469def4.png)
![image](https://user-images.githubusercontent.com/66349613/215368503-2b910068-60e7-46ce-8b20-a32591cc0ba9.png)

**Comment by hotlong** [comment]
要看一下这个字段的定义，和其他lookup有什么不同。

**Reply by daiyaozuo** [reply]
这个字段的类型在3000和5000都是相关表的类型，都是同一套代码而且连接的数据库也是同一个，在5000可以正常回显名称，在3000有时会回显id，这个是3000项目不兼容么
